sdk/communication/Azure.Communication.Administration/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -0,0 +1,471 @@
+            var reservationPurchaseOperation = client.StartPurchaseReservation(reservationId);
+
+            while (true)
+            {
+                reservationPurchaseOperation.UpdateStatus();
+                if (reservationPurchaseOperation.HasCompleted)
+                {
+                    break;
+                }
+
+                Thread.Sleep(1000);
+            }
+            #endregion Snippet:StartPurchaseReservation
+
+            #region Snippet:ListAcquiredPhoneNumbers
+            var acquiredPhoneNumbers = client.GetAllPhoneNumbers(locale);
+
+            foreach (var phoneNumber in acquiredPhoneNumbers)
+            {
+                Console.WriteLine($"Phone number: {phoneNumber.PhoneNumber}, activation state: {phoneNumber.ActivationState}");
+            }
+            #endregion Snippet:ListAcquiredPhoneNumbers
+
+            var acquiredPhoneNumber = reservationOperation.Value.PhoneNumbers.First();
+
+            #region Snippet:ReleasePhoneNumbers
+            //@@var acquiredPhoneNumber = "<acquired_phone_number>";
+            var phoneNumbers = new List<PhoneNumber> { new PhoneNumber(acquiredPhoneNumber) };
+            var phoneNumberReleaseOperation = client.StartReleasePhoneNumbers(phoneNumbers);
+
+            while (true)
+            {
+                phoneNumberReleaseOperation.UpdateStatus();
+                if (phoneNumberReleaseOperation.HasCompleted)
+                {
+                    break;
+                }
+
+                Thread.Sleep(1000);
+            }
+            #endregion Snippet:ReleasePhoneNumbers
+        }
+
+        [Test]
+        [SyncOnly]
+        public void GetAllSupportedCountries()
+        {
+            var client = CreateClient(false);
+            var locale = "en-US";
+
+            #region Snippet:GetAllSupportedCountries
+            var supportedCountries = client.GetAllSupportedCountries(locale);
+            foreach (var country in supportedCountries)
+            {
+                Console.WriteLine($"Supported country code: {country.CountryCode}, name: {country.LocalizedName}");
+            }
+            #endregion Snippet:GetAllSupportedCountries
+        }
+
+        [Test]
+        [AsyncOnly]
+        public async Task GetAllSupportedCountriesAsync()
+        {
+            var client = CreateClient(false);
+            var locale = "en-US";
+
+            #region Snippet:GetAllSupportedCountriesAsync
+            var supportedCountries = client.GetAllSupportedCountriesAsync(locale);
+            await foreach (var country in supportedCountries)
+            {
+                Console.WriteLine($"Supported country code: {country.CountryCode}, name: {country.LocalizedName}");
+            }
+            #endregion Snippet:GetAllSupportedCountriesAsync
+        }
+
+        [Test]
+        [SyncOnly]
+        public void GetPhonePlanLocationOptions()
+        {
+            var client = CreateClient(false);
+            var locale = "en-US";
```

Comment:
`const` instead of `var`. Same for the country code and other methods

Reply:
Done